src/server/plugins/engine/pageControllers/QuestionPageController.ts

```diff
@@ -261,10 +261,11 @@ export class QuestionPageController extends PageController {
       context: FormContext,
       h: Pick<ResponseToolkit, 'redirect' | 'view'>
     ) => {
-      const { model, viewName } = this
+      const { collection, model, viewName } = this
       const { evaluationState, state } = context
 
       const viewModel = this.getViewModel(request, context)
+      viewModel.errors = collection.getErrors(viewModel.errors)
```

Comment:
We already do this on POST

Without it, state errors from previous pages will appear when `?force` arrives in https://github.com/DEFRA/forms-runner/pull/646